historical/common/proxy.py

```diff
@@ -0,0 +1,110 @@
+"""
+.. module: historical.common.proxy
```

Comment:
This is kinda weird place to have a `lambda_handler` as by definition it wouldn't be a common function?

Reply:
It is a common function that should apply to all technologies.